src/style.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * create a style tag and append to head tag
+ * @params {String} css style
+ */
+
+function insertStyle ( css ) {
+    if(!css) return ;
+
+    if(typeof(window) == 'undefined') return ;
+    let style = document.createElement('style');
+    style.setAttribute('media', 'screen');
+
+    style.innerHTML = css
+    document.head.appendChild(style);
+    return css
+}
```

Comment:
looks like this project uses semicolons, so you might want to add them in everywhere in this PR... also I'd change the spacing to match the project (space after `if`, no spaces in param list parens, etc).